react-components/src/architecture/concrete/observationsDomainObject/ObservationsView.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { Box3 } from 'three';
+import { type ObservationsDomainObject } from './ObservationsDomainObject';
+import { GroupThreeView } from '../../base/views/GroupThreeView';
+import { type Overlay3DCollection } from '@cognite/reveal';
+import { type Observation } from './models';
+
+export class ObservationsView extends GroupThreeView<ObservationsDomainObject> {
+  private readonly _collection: Overlay3DCollection<Observation>;
```

Comment:
Please follow the example code. You should not store the data here, but get it by the DomainObject.